packages/select-rich/test/lion-select-rich.test.js

```diff
@@ -0,0 +1,193 @@
+import { expect, fixture, html } from '@open-wc/testing';
+
+import '../lion-select-rich.js';
+
+describe('lion-select-rich', () => {
+  it('has an empty modelValue by default', async () => {
+    const el = await fixture(html`
+      <lion-select-rich>
+        <lion-listbox slot="input">
+          <lion-option value="nr1">Item 1</lion-option>
+          <lion-option value="nr2">Item 2</lion-option>
+        </lion-listbox>
+      </lion-select-rich>
+    `);
+    expect(el.modelValue).to.equal('');
+  });
+
+  it('has the selected option as modelValue', async () => {
+    const el = await fixture(html`
+      <lion-select-rich>
+        <lion-listbox slot="input">
+          <lion-option value="nr1" selected>Item 1</lion-option>
+          <lion-option value="nr2">Item 2</lion-option>
+        </lion-listbox>
+      </lion-select-rich>
+    `);
+    expect(el.modelValue).to.equal('nr1');
+  });
+
+  it('can preselect an option', async () => {
+    const el = await fixture(html`
+      <lion-select-rich .modelValue="${'nr2'}">
+        <lion-listbox slot="input">
+          <lion-option value="nr1">Item 1</lion-option>
+          <lion-option value="nr2">Item 2</lion-option>
+        </lion-listbox>
+      </lion-select-rich>
+    `);
+    expect(el._listboxElement.value).to.equal('nr2');
+  });
+
+  it('shows content of the selected option inside the invoker', async () => {
+    const el = await fixture(html`
+      <lion-select-rich>
+        <lion-listbox slot="input">
```

Comment:
I think calling this slot "input" might lead to our users conflating it with the invoker, because I imagine the invoker looks similar to an input field, and it's where the selected options ends up in.

Suggestion: call it `options`?

Reply:
Yeah, though to find the right compromise between AD and SC. Hopefully this docs answers it: https://github.com/ing-bank/lion/blob/feat/richSelect/packages/select-rich/docs/DesignConsiderations.md